chain/client/src/chunk_validation.rs

```diff
@@ -421,10 +407,54 @@ fn apply_result_to_chunk_extra(
     )
 }
 
+fn send_chunk_endorsement_to_block_producers(
```

Comment:
We could name it `endorse_chunk` for demonstrativeness. Not a strong opinion

Reply:
I like the current name as it's more explicit